### PR TITLE
First tutorial page: getting started with GCE

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

**Comment by dchen1107** [comment]
cc/ @dchen1107

File: docs/tutorial/gce.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+Make sure you have [installed kops](../install.md) and [installed kubectl](../install.md), and installed
+the [gcloud tools](https://cloud.google.com/sdk/downloads).
+
+You'll need a Google Cloud account, and make sure that gcloud is logged in to your account using `gcloud init`.
+
+You should confirm that basic commands like `gcloud compute zones list` are working.
+
+You'll also need to [configure default credentials](https://developers.google.com/accounts/docs/application-default-credentials), using `gcloud auth application-default login`.
+
+<!-- TODO: Can we get rid of `gcloud auth application-default login` ? -->
```
Comment (chrislovecnm) [inline docs/tutorial/gce.md]: Not that I have found with the API

**Comment by chrislovecnm** [comment]
Yay docs

**Pull request merged**